pkg/unvanquished_src.dpkdir/ui/options_ui.rml

```diff
@@ -55,5 +55,30 @@
 			<input cvar="cg_drawFPS" type="checkbox" />
 			<p><translate>Frames per second.</translate></p>
 		</row>
+		<row>
+			<h3><translate>Console colors</translate></h3>
+			<p>
+				<row>
+					<input type="range" min="0" max="1" step="0.05" cvar="con_colorRed"/>
+					<translate>red:</translate>&nbsp;<inlinecvar cvar="con_colorRed" type="number" format="%0.2f"/>
```

Comment:
UI text should start with capital letters to be consistent with the rest of the menu